# Generated by Django 3.2.4 on 2021-07-07 02:01

from django.conf import settings
import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='Verify',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('order_number', models.CharField(help_text='Please scan the tray id.', max_length=10, validators=[django.core.validators.MinLengthValidator(10, 'Please scan the tray id.')])),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('weight', models.DecimalField(decimal_places=2, help_text='Measured in oz.', max_digits=6)),
                ('upc', models.BigIntegerField(help_text='Please scan the UPC.')),
                ('qty', models.BigIntegerField(help_text='Please enter the quantity.')),
                ('warehouse', models.CharField(help_text='Please enter the warehouse.', max_length=2, validators=[django.core.validators.MinLengthValidator(2, 'Please enter the warehouse id.')])),
                ('location', models.CharField(help_text='Please enter the location sloted.', max_length=10, validators=[django.core.validators.MinLengthValidator(10, 'Please scan the location.')])),
                ('owner', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
